packages/cadl-ranch-specs/http/payload/multipart/main.tsp

```diff
@@ -38,4 +70,177 @@ Content-Type: application/octet-stream;
   @post
   @route("/mixed-parts")
   op basic(@header contentType: "multipart/form-data", @body body: MultiPartRequest): NoContentResponse;
+
+  @scenario
+  @scenarioDoc("""
+Expect request:
```

Comment:
Is it possible to verify this payload is matching the real service's payload?

We have 4 services now as you documented [here](https://loop.microsoft.com/p/eyJ3Ijp7InUiOiJodHRwczovL21pY3Jvc29mdC5zaGFyZXBvaW50LmNvbS8%2FbmF2PWN6MGxNa1ltWkQxaUlXMVFSQzFzTkZWeFgydFRVblJqVXpSbmQwZDZRazl3TFhGWWNHcHVNbkJCZFU5cVRXMVlOR2Q0YkZrellrZ3lWRWxmUkZkU1luRllhRXRqUkROcVgwUW1aajB3TVZSS1dGbFZWMU5QVmtSRVNrdEJRa3hTTlVOTE0waFBVbEpNVWxWSE1rcFJKbU05Sm1ac2RXbGtQVEUlM0QiLCJyIjpmYWxzZX0sInAiOnsidSI6Imh0dHBzOi8vbWljcm9zb2Z0LnNoYXJlcG9pbnQuY29tLzpmbDovci9zaXRlcy9iMWVmYzE4ZS1lZTZlLTRkN2YtODIzZS05MzZkOTAxNTMyMzAvU2hhcmVkJTIwRG9jdW1lbnRzL0xvb3BBcHBEYXRhL1VudGl0bGVkJTIwcGFnZSUyMDIubG9vcD9kPXcwYjU1MTg0MzNiZGY0ODk0OTNjYTU2Nzg2YjRmMDUyZiZjc2Y9MSZ3ZWI9MSZuYXY9Y3owbE1rWnphWFJsY3lVeVJtSXhaV1pqTVRobExXVmxObVV0TkdRM1ppMDRNak5sTFRrek5tUTVNREUxTXpJek1DWmtQV0loYlZCRUxXdzBWWEZmYTFOU2RHTlROR2QzUjNwQ1QzQXRjVmh3YW00eWNFRjFUMnBOYlZnMFozaHNXVE5pU0RKVVNWOUVWMUppY1Zob1MyTkVNMnBmUkNabVBUQXhWRXBZV1ZWWFUwUkVRa3RSV0ZoYU0xTlNSVXBJVTFOWFVFSldWVFpDU2xBbVl6MGxNa1ltWm14MWFXUTlNU1poUFV4dmIzQkJjSEFtY0QwbE5EQm1iSFZwWkhnbE1rWnNiMjl3TFhCaFoyVXRZMjl1ZEdGcGJtVnlKbmc5SlRkQ0pUSXlkeVV5TWlVelFTVXlNbFF3VWxSVlNIaDBZVmRPZVdJelRuWmFibEYxWXpKb2FHTnRWbmRpTW14MVpFTTFhbUl5TVRoWmFVWjBWVVZSZEdKRVVsWmpWamx5VlRGS01Ga3hUVEJhTTJSSVpXdEtVR05ETVhoWFNFSnhZbXBLZDFGWVZsQmhhekYwVjBSU2JtVkhlRnBOTWtwSlRXeFNTbGd3VWxoVmJVcDRWMGRvVEZrd1VYcGhiRGxGWmtSQmVGWkZjRmxYVmxaWVZUQTVWMUpGVWt0VE1FWkRWRVpKTVZFd2MzcFRSVGxUVld0NFUxWlZZM2xUYkVVbE0wUWxNaklsTWtNbE1qSnBKVEl5SlROQkpUSXlNVEJoWlRBMk0yUXRZV0poWlMwME5UVXdMV0ZoWWpNdE1qWXhNR1U0TVdNME5XRXlKVEl5SlRkRSIsInIiOmZhbHNlfSwiaSI6eyJpIjoiMTBhZTA2M2QtYWJhZS00NTUwLWFhYjMtMjYxMGU4MWM0NWEyIn19).

Reply:
We can't know how service parse payload sent by client. But as long as it follows HTTP guideline, it shall pass the cadl-ranch case. (now Java/Python can)